Engine/Results/BacktestingResultHandler.cs

```diff
@@ -509,6 +514,30 @@ public virtual void RuntimeError(string message, string stacktrace = "")
             _errorMessage = message;
         }
 
+        /// <summary>
+        /// Samples portfolio equity, benchmark, and daily performance
+        /// </summary>
+        /// <param name="time">Current UTC time in the AlgorithmManager loop</param>
+        /// <param name="force">Force sampling of equity, benchmark, and performance to be </param>
+        public override void Sample(DateTime time, bool force = false)
+        {
+            // Only sample once a day unless forced
+            var dayChanged = PreviousUtcSampleTime.Date != time.Date;
+            if (dayChanged || force)
+            {
+                // Use previous time for sample unless forced
+                var sampleTime = force ? time : PreviousUtcSampleTime;
+
+                // Sample strategy capacity, round to 1k
+                var roundedCapacity = Math.Round(_capacityEstimate.Capacity.Value / 1000) * 1000;
+                Sample("Capacity", "Strategy Capacity", 0, SeriesType.Line, sampleTime,
+                    roundedCapacity, AlgorithmCurrencySymbol);
```

Comment:
Why special behavior for Capacity? maybe if just backtesting we can have SampleCapacity which is `nop` in live trading?